requirements.txt

```diff
@@ -0,0 +1 @@
+cfn-lint
```

Comment:
may eventually consider pinning the version here, this repo looks like a good fit for dependabot

Reply:
For sure.